Fix disabled `"Add WMS Layers Panel"` after changing projection of current selected layer

volterra79 opened the pull request:
Closes: #211

Fix misspelling **loading** attribute of a WMSLayersPanel.vue component.
Fix issue of no layers in select layer when all selected layers are removed

![Screenshot from 2022-09-27 12-06-12](https://user-images.githubusercontent.com/1051694/192498047-776b0b6e-a02b-42c7-a7af-b6418bcd8d90.png)

![Screenshot from 2022-09-27 12-04-48](https://user-images.githubusercontent.com/1051694/192497768-3ba8d89c-6bb6-4af3-b5a5-3d921c8e72e1.png)

volterra79 committed:
Fix issue on disabled Add WMS Layer Panel when change projection on current selected layer

Raruto commented on src/components/WMSLayersPanel.vue:
```diff
@@ -71,25 +71,21 @@ export default {
         if (layers.length === 1) {
           this.epsg = this.layerProjections[firstLayer].crss[0];
-          this.projections = this.layerProjections[firstLayer].crss;
+          this.projections = [...this.layerProjections[firstLayer].crss];
```
@volterra79 Can you add comment above this line? (eg. shallow copy required due to ...)

volterra79 replied:
@Raruto i add new commit without shallow copy

volterra79 committed:
Remove shallow copy of first selected layer crss.
It use a initial crss array of crs (epsg) supported by first select layer.
When all selected layers is removed, no this.projections.splice(0) but this.projections = []. In this way first selected layer crss array is touch.
